integration-tests/api/__tests__/admin/product.js

```diff
@@ -909,6 +909,93 @@ describe("/admin/products", () => {
     })
   })
 
+  describe("GET /admin/products/:id", () => {
+    const productId = "testing-get-product"
+
+    beforeEach(async () => {
+      await simpleProductFactory(dbConnection, {
+        id: productId,
+        variants: [
+          {
+            title: "Test variant",
+            prices: [
+              {
+                currency: "usd",
+                amount: 100,
+              },
+            ],
+          },
+        ],
+      })
+
+      await adminSeeder(dbConnection)
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("should get a product", async () => {
+      const api = useApi()
+
+      const res = await api
```

Comment:
comment: in this test, if no expand are provided, by default the variants and prices will be part of the relations. Should we check that they are all present to prevent future issues? at least that they are all defined